example/custom-config/server/main.go

```diff
@@ -20,13 +20,12 @@ import (
 	"context"
 	"log"
 
-	consulapi "github.com/hashicorp/consul/api"
-
+	"github.com/cloudwego/kitex-examples/hello/kitex_gen/api"
```

Comment:
Do not use this package， instead of `github.com/kitex-contrib/registry-consul/example/hello`

Reply:
done